modules/net-lb-app-ext-regional/health-check.tf

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Health check resource.
+
```

Comment:
is there a reason why this file substantially differs from the health check definitions in the other LBs? eg https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/f89c72b3d3ed5ae11974ff902b0500125123ea7b/modules/net-lb-ext/health-check.tf

Reply:
Good point. There's some discrepancy. Let's merge this and I'll send another PR aligning all the HCs across all LB modules.